### ISSUE TITLE
A question about Session::prepareCommonDownload()

### ISSUE THREAD

**Issue by L-Super** [libcpr/cpr#1138]
### Description

I noticed that in the [Session::prepareCommonDownload()](https://github.com/libcpr/cpr/blob/ffef58eddea31cdf46a46c6ac0f14138ade76176/cpr/session.cpp#L244) function, `prepareCommonShared()` is called, then calls `prepareHeader()`. However, `prepareHeader()` is also called within `prepareCommonShared()`. Is this a duplicate call?

### Example/How to Reproduce

1. Create a `cpr::Session`
2. Set option ...
3. Perform the request

### Where did you get it from?

GitHub (branch e.g. master)

### Additional Context/Your Environment

- OS:
- Version:

**Comment by COM8** [comment]
Yes, this is a duplicate call. Do you want to create a PR to remove it?

Thanks for reporting!

**Comment by L-Super** [comment]
Sure! I will submit a PR.



---Original---
From: "Fabian ***@***.***&gt;
Date: Sat, Nov 9, 2024 14:02 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [libcpr/cpr] A question about Session::prepareCommonDownload()(Issue #1138)




 
Yes, this is a duplicate call. Do you want to create a PR to remove it?
 
Thanks for reporting!
